change/@office-iss-react-native-win32-2393f54c-703d-4a15-bb87-43de8cb83d42.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
```

Comment:
These change files don't look right - the `type` should be `prerelease`, `dependentChangeType` should be `patch`, and the comment should probably be `Integrate RN Nightly Build 0.80.0-nightly-20250317-fc7385c1e`